spec/services/nomis/elite2/offender_api_spec.rb

```diff
@@ -115,11 +115,10 @@
       image_id = 1_340_556
       uri = "https://gateway.t3.nomis-api.hmpps.dsd.io/elite2api/api/images/#{image_id}/data"
 
-      WebMock.stub_request(:get, uri).to_raise(
-        Faraday::ResourceNotFound.new('error', status: 404)
-      )
+      stub_request(:get, uri).to_return(status: 404)
 
-      expect{ described_class.get_image(booking_id) }.to raise_error(Faraday::ResourceNotFound)
+      response = described_class.get_image(booking_id)
+      expect(response).not_to be nil?
```

Comment:
can we not check for default_image instead?

Reply:
Probably, we'd have to load the bytes from the local image and check them, but shouldn't be too hard.